docs/api.rst

```diff
@@ -2101,42 +2101,6 @@ Request creation of oracle price cache
    :statuscode 500: Internal rotki error
    :statuscode 502: The oracle could not be queried due to an error on their side.
 
-Delete an oracle price cache
```

Comment:
This makes no sense. I did not mean to do it now. I meant to do it perhaps at some other point. This endpoint already exists and is used with other verbs. This change you are doing does not make sense then as is. Please ask before goign and doing stuff like that.